### PR TITLE
Lr

### PR DESCRIPTION
**Pull request by Vidhan**

### CODE REVIEW

**Comment by murphp15** [comment]
What does LR stand for? 
Could you give a small description for the PR.

File: allennlp/data/dataset_readers/reading_comprehension/squad.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ def _read(self, file_path: str):
             dataset = dataset_json['data']
         logger.info("Reading the dataset")
         for article in dataset:
+            #import pdb; pdb.set_trace()
```
Comment (murphp15) [inline allennlp/data/dataset_readers/reading_comprehension/squad.py]: Did you mean to leave this in here?

File: scripts/CMU/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+        pass
+
+    def _word_cleanup(self, lookups):
+        result = []
+
+        for word in lookups:
+            try:
+                assert word[0] == '['
+            except AssertionError as e:
+                print word
```
Comment (murphp15) [inline scripts/CMU/preprocess.py]: I think this should be print(word) to make in compatible with python 3.

**Comment by schmmd** [comment]
Hi @Vidhan, I'm glad to see you're using AllenNLP!  While we welcome contributions, we need a bit more context before we can even start to review a PR.  Can you add a detailed description that explains the contribution if you think this contribution would be a benefit to AllenNLP?

You'll find our [Contributing Guidelines](https://github.com/allenai/allennlp/blob/master/CONTRIBUTING.md) to be a good start for guidance about contributions.  Feel free to reach out if you have any other questions.

File: scripts/CMU/knowledge_graph.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        for src in range(node_len):
+            for dst in range(node_len):
+                if dist[src][dst] != sys.maxsize:
+                    print((sorted_nodes[src], sorted_nodes[dst], dist[src][dst]))
+
+
+
+if __name__ == "__main__":
+    kg = KnowledgeGraph()
+    path = "/Users/prasoon/CMU/10608/pt-allennlp/scripts/CMU/meeting.with_hints"
```
Comment (murphp15) [inline scripts/CMU/knowledge_graph.py]: remove reference to own machine.

**Pull request closed without merging**